generated/provider_dependencies.json

```diff
@@ -846,6 +846,7 @@
     "plugins": [],
     "cross-providers-deps": [
       "amazon",
+      "common.compat",
```

Comment:
why we need this change here?

Reply:
We are using compat provider for imports to work with old core location. so here in this case msgraph uses timedelta, https://github.com/apache/airflow/pull/43571/files/a750af6fec2a04ba5a347303fb4397a4e496d56e#diff-af3b61e0f2e76a9841ce87162351100deee4bc736a90d4cb8715acd9fb61473cR23 

pre-commit generated this with common.compat as cross provider